libs/designsystem/accordion/src/accordion-item.component.scss

```diff
@@ -41,7 +41,9 @@ $padding: utils.size('s');
 }
 
 .content-body {
-  padding: 0 $padding $padding;
+  &.padding {
+    padding: 0 $padding $padding;
+  }
 }
```

Comment:
Now that there is only one style rule, consider rewriting as .content-body.padding for readability